week-2-wide-perspective/w02e05-simple-server-side/src/types/enviroments/enviroments.d.ts

```diff
@@ -0,0 +1,9 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      DATABASE_URL: string;
+      ENV: "test" | "dev" | "prod";
+    }
+  }
+}
+export {};
```

Comment:
super że dodane. 
Ja bym to nazwał `env.d.ts` - wtedy widać którego pliku dokładnie dotyczy